packages/swap/prisma/schema.prisma

```diff
@@ -419,19 +421,21 @@ model StateChainError {
 }
 
 model Quote {
-  id                         Int                @id @default(autoincrement())
-  swapDepositChannel         SwapDepositChannel @relation(fields: [swapDepositChannelId], references: [id])
-  swapDepositChannelId       BigInt             @unique
-  srcAsset                   InternalAsset
-  destAsset                  InternalAsset
+  id                      Int                @id @default(autoincrement())
+  swapDepositChannel      SwapDepositChannel @relation(fields: [swapDepositChannelId], references: [id])
+  swapDepositChannelId    BigInt             @unique
+  swapRequest             SwapRequest?       @relation(fields: [swapRequestId], references: [id])
+  swapRequestId           BigInt?            @unique
+  srcAsset                InternalAsset
+  destAsset               InternalAsset
   maxBoostFeeBps             Int?
   numberOfChunks             Int?
-  depositAmount              Decimal            @db.Decimal(30, 0)
-  intermediateAmount         Decimal?           @db.Decimal(30, 0)
-  egressAmount               Decimal            @db.Decimal(30, 0)
-  estimatedPrice             Decimal
-  executedPrice              Decimal?
-  executedSlippagePercentage Decimal?
+  depositAmount           Decimal            @db.Decimal(30, 0)
+  intermediateAmount      Decimal?           @db.Decimal(30, 0)
+  egressAmount            Decimal            @db.Decimal(30, 0)
+  estimatedPrice          Decimal
+  executedPrice           Decimal?
+  executedSlippagePercent Decimal?
 
```

Comment:
since we're storing all the info needed for reporting here, maybe we can also store `quotedAt` and `executedAt` timestamps to know if someone used the channel late

Reply:
thats a good idea! unfortuately we dont have a `quotedAt` at the moment because there is no timestamp on the quote, but ill think about how to do this